lib/bundler/cli/plugin.rb

```diff
@@ -19,5 +19,10 @@ class CLI::Plugin < Thor
     def install(*plugins)
       Bundler::Plugin.install(plugins, options)
     end
+
+    desc "uninstall PLUGINS", "Uninstall the plugins"
```

Comment:
Is `PLUGINS` an optional param or do I need to list them?

My mental model when I see this command is to associate it as the opposite to `bundle install` for gems, as such it's not a natural naming convention. It would be better to associate this command more to `bundle plugins` like `bundle plugins uninstall` or `bundle plugins --uninstall` or something that tells the user this is for plugins and not gems. Making this distinction is important for good UX of bundler.

The short description can be improved like "Uninstalls the specified plugins".

The long description "Uninstall" needs to be "Uninstalls". Can you elaborate what happens if no `PLUGINS` is specified, i.e. will it remove all plugins? Is a list of plugins required? Anything else that a user should know about this command.